.aptly.conf

```diff
@@ -0,0 +1,36 @@
+{
+  "rootDir": "$HOME/.aptly",
```

Comment:
not sure if it makes any difference when running in the pipeline regarding the $HOME variable, but if you have tested it before then all good!

Reply:
Good point. I didn't test it in the pipeline yet, but this is the default value (coming from [here](https://www.aptly.info/doc/configuration/))